packages/gluestick-generators/src/requireGenerator.js

```diff
@@ -8,14 +8,17 @@ const { convertToCamelCase, convertToKebabCase } = require('./utils');
 // $FlowFixMe
 module.createTemplate = createTemplate;
 
-const PREDEFINED: string = 'predefined';
+const PATH_TO_GLUESTICK_TEMPLATES = '../../gluestick/generator/predefined';
```

Comment:
In that case extract path to gluestick templates to `getPossiblePaths` and `requireGeneratorConfig` argument and pass it from `generator.js`.